test/Form_test.js

```diff
@@ -70,14 +69,29 @@ describe("Form", () => {
     const formData = {foo: "invalid"};
 
     function FieldTemplate(props) {
-      const {id, classNames, label, help, required, description, errors, children} = props;
+      const {
+        id,
+        classNames,
+        label,
+        help,
+        rawHelp,
+        required,
+        description,
+        rawDescription,
+        errors,
+        rawErrors,
+        children,
+      } = props;
       return (
         <div className={"my-template " + classNames}>
           <label htmlFor={id}>{label}{required ? "*" : null}</label>
           {description}
+          <span id={`${id}__raw_description`}>{`${rawDescription} rendered from the raw format`}</span>
```

Comment:
Nit: We should stick to a single id format here I think, `${id}__raw_description` is fine.